tests/unit/s2n_mem_usage_test.c

```diff
@@ -140,6 +139,20 @@ int main(int argc, char **argv)
     ssize_t vm_data_after_handshakes = get_vm_data_size();
     EXPECT_NOT_EQUAL(vm_data_after_handshakes, -1);
 
+    for (int i = 0; i < CONNECTIONS; i++) {
+        EXPECT_SUCCESS(s2n_connection_free_handshake(servers[i]));
+        EXPECT_SUCCESS(s2n_connection_free_handshake(clients[i]));
+    }
+    ssize_t vm_data_after_free = get_vm_data_size();
+    EXPECT_NOT_EQUAL(vm_data_after_free, -1);
+
+    for (int i = 0; i < CONNECTIONS; i++) {
+        EXPECT_SUCCESS(s2n_connection_release_buffers(servers[i]));
```

Comment:
Do we need a test that we're still able to resume writes and reads after releasing buffers?

Reply:
good idea.  will add that to the self-talk test.